src/Calendar.js

```diff
@@ -200,6 +200,15 @@ class Calendar extends React.Component {
     */
    onSelectEvent: PropTypes.func,
 
+   /**
+    * Callback fired when a calendar availability is clicked.
+    *
+    * ```js
+    * function(availability: object, e: SyntheticEvent)
+    * ```
+    */
+   onAvailabilityClick: PropTypes.func,
```

Comment:
Can we call this `onSelectAvailability` to be consistent? There's already `onSelectEvent` and `onSelectSlot`. It's a little confusing to start using "click" instead of "select" just for availabilities.